plugins/org.eclipse.n4js/src/org/eclipse/n4js/resource/N4JSResource.java

```diff
@@ -834,6 +924,10 @@ protected EObject getEObjectForURIFragmentRootSegment(String uriFragmentRootSegm
 				if (position < contents.size() && position >= 1) {
 					return contents.get(position);
 				}
+				if (position == 0 && contents.size() > 1) {
+					// we have a proxified AST, but a TModule loaded from the index
+
+				}
```

Comment:
Nothing is done here?

Reply:
Yes, that slipped through. The condition is save to be removed again.